CrisisTextLine/Elmer
# Auto force melt

#### What's this PR do?
Completes the Auto Force Melt Process from https://github.com/CrisisTextLine/Elmer/pull/283
#### Where should the reviewer start?
I started changing things from warnOrMelt() in the above PR - overwrote some of my commit history so I'm ditching the other branch
#### How should this be manually tested?
Change the email in mergeDataForSending() and run autoMeltCanvasUsers(). 
Check that you receive the appropriate emails
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/secure/RapidBoard.jspa?rapidView=12&view=detail&selectedIssue=AT-13&quickFilter=44
#### Questions:
